deploy/postgres/postgres-deployment-ephemeral.yaml

```
@@ -15,7 +15,7 @@ spec:
     spec:
       containers:
         - name: postgres
-          image: quay.io/ocpmetal/postgresql-12-centos7
+          image: quay.io/edge-infrastructure/postgresql-12-centos7
```

Any chance to change to the upstream project?
We're already using it in podman job, so it should behave well
```suggestion
          image: quay.io/centos7/postgresql-12-centos7
```